packages/angular/projects/ui-angular/src/lib/primitives/checkbox/checkbox.component.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { CheckboxComponent } from './checkbox.component';
+
+describe('CheckboxComponent', () => {
+  let fixture: CheckboxComponent;
+
+  beforeEach(() => {
+    fixture = new CheckboxComponent();
+  });
+  it('Default checkbox behaviour -> ', () => {
+    expect(fixture.isChecked).toEqual(false);
+  });
+});
```

Comment:
This is a great first start to adding in testing!

Could you also add in the testing library? https://testing-library.com/docs/angular-testing-library/intro

And convert these tests to use that instead? That way we have consistency between all 3 platforms.

Reply:
Ok, I will look into this.

Reply:
Same comment as @ErikCH